src/Service/Mailer/Mail/Account/AccountTransferMailer.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Service\Mailer\Mail\Account;
+
+use App\Entity\User;
+use App\Service\Mailer\Mail\AbstractNotificationMailer;
+use App\Service\Mailer\NotificationMail;
+use App\Service\Mailer\NotificationMailerType;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Security\Http\LoginLink\LoginLinkHandlerInterface;
+
+class AccountTransferMailer extends AbstractNotificationMailer
+{
+    protected ?NotificationMailerType $mailerType = NotificationMailerType::TYPE_ACCOUNT_TRANSFER;
+    protected ?string $mailerSubject = 'Transfert de votre compte Histologe';
+    protected ?string $mailerTemplate = 'transfer_account_email';
+
+    public function __construct(
+        protected MailerInterface $mailer,
+        protected ParameterBagInterface $parameterBag,
+        protected LoggerInterface $logger,
+        protected UrlGeneratorInterface $urlGenerator,
+        private readonly LoginLinkHandlerInterface $loginLinkHandler,
+    ) {
+        parent::__construct($this->mailer, $this->parameterBag, $this->logger, $this->urlGenerator);
+    }
+
+    public function getMailerParamsFromNotification(NotificationMail $notificationMail): array
+    {
+        $user = $notificationMail->getUser();
+        $loginLinkDetails = $this->loginLinkHandler->createLoginLink($user);
+        $loginLink = $loginLinkDetails->getUrl();
+
+        $link = User::STATUS_ACTIVE === $user->getStatut() ?
```

Comment:
erreur au transfert de l'utilisateur : App\Manager\UserManager::transferUserToPartner(): Argument #1 ($user) must be of type App\Entity\User, null given, called in /app/src/Controller/Back/BackPartnerController.php on line 157

Reply:
C'est corrigé

Reply:
toujours l'erreur

Reply:
doublon d'erreur déjà testée.